Source/Scene/GlobeSurfaceTile.js

```diff
@@ -318,7 +318,7 @@ define([
 
         // If this tile's terrain and imagery are just upsampled from its parent, mark the tile as
         // upsampled only.  We won't refine a tile if its four children are upsampled only.
-        var isUpsampledOnly = defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
+        var isUpsampledOnly = false;//defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
```

Comment:
We can not change the default behaviour of Cesium.
Maybe we can mark the tiles we skipped because the available tiles at its level was empty,
and then change the condition.